crates/js-component-bindgen/src/transpile_bindgen.rs

```diff
@@ -458,10 +458,24 @@ impl<'a> Instantiator<'a, '_> {
             self.exports.insert(name.to_string(), key.clone());
         }
 
+        // Determine the package
+        let package = self
+            .resolve
+            .packages
+            .get(
+                self.resolve.worlds[self.world]
+                    .package
+                    .expect("unexpectedly missing package in world"),
+            )
+            .expect("unexpectedly missing package in world for ID");
+        let package_version = package.name.version.as_ref();
+
+        // Iterate & generate exports
```

Comment:
This code isn't doing export generation, it is doing a pre-pass to populate resource data structures.

Reply:
Ahh! Thank you -- I actually totally forgot my own `TODO` in the that other bit of the file!